flocker/acceptance/test_api.py

```diff
@@ -462,12 +462,13 @@ def in_current():
         return creating
 
 
-def create_dataset(test_case):
+def create_dataset(test_case, maximum_size=REALISTIC_BLOCKDEVICE_SIZE):
     """
     Create a dataset on a single-node cluster.
 
     :param TestCase test_case: The test the API is running on.
-
+    :param int maximum_size: The size of the dataset to create on the test
+        cluster. Defaults to ``REALISTIC_BLOCKDEVICE_SIZE``.
```

Comment:
Any sensible API documentation generation tool should provide information about the default automatically.